src/main/host/descriptor/udp.c

```diff
@@ -24,12 +24,36 @@
 #include "main/utility/utility.h"
 #include "support/logger/logger.h"
 
+enum UDPState {
```

Comment:
I used this format in order to keep the same structure as `tcp.c`. Should I update the enums and prefixes for file-scoped symbols in `tcp.c` as well?

Reply:
Oh, that's surprising. It looks like Rob wrote it that way, and that's who I thought established/preferred the other convention. I'm fine with leaving as-is (though maybe @robgjansen will want to chime in?)

Reply:
Looks like I wrote most of those non-typedef'd TCP enums 9 years ago. My guess is that I didn't realize you could typedef enums at the time :-)

I don't like writing out the `struct` and `enum` prefixes all the time. Also, typedef'ing structs is useful for modularization, because you can hide the struct definition inside the implementation c file. Obviously that's not relevant to objects that are only ever defined and used at the file level, though, but being consistent in both cases is nice.

Maybe a compromise would be to always typedef all structs, but don't typedef enums? That way if you don't see `enum` you know it's a `struct`.

In any case though, this is one of those bike shed issues and I don't think it's really worth going through and fixing this everywhere in this PR. We can discuss further if we think the issue is causing enough confusion/annoyance to warrant effort on refactoring rather than just replacing with Rust code ;)

Reply:
Okay thanks, leaving for now, but I did add the `_` prefix as discussed below.